src/theme/components/heading.ts

```diff
@@ -52,46 +52,88 @@ export const headings = {
     lineHeight: "1.10",
     width: "auto",
     a: { textUnderlineOffset: "7px" },
+    "@media (min-width: 0px)": {
+      fontSize: "mobile.heading.display1",
+    },
+    "@media (min-width: 600px)": {
+      fontSize: "desktop.heading.display1",
+    },
```

Comment:
I'm not sure if you need the media query for "0px" because that's the default. So it should be
```
fontSize: "mobile.heading.display1",
"@media (min-width: 600px)": {
  fontSize: "desktop.heading.display1",
},
```

Reply:
Yes, I thought that as well.  I gave it a try at one point, but the default styles on the Chakra `Heading` seem to override those styles.  However, when I put it into the media query, it worked.